src/index.js

```diff
@@ -1,139 +1,27 @@
-        const values = []
-        const fields = [basicQuery]
-        if (req.query.category) {
-            fields.push('category=?')
-            values.push(req.query.category)
-        }
-        if (req.query.modelName) {
-            fields.push('modelName=?')
-            values.push(req.query.modelName)
-        }
-        if (req.query.registrationNumber) {
-            fields.push('registrationNumber=?')
-            values.push(req.query.registrationNumber)
-        }
-        const sqlQuery = fields.join(' AND ')
-        connection.query(sqlQuery, values, (err, result) => {
-            if (err) throw err;
-            res.status(200).json(result);
-        });
-    } catch (e) {
-        console.log(e.message)
-        res.status(500).json({ errorMessage: 'An unexpected error occured. Check server logs' });
-    }
-})
-//update
-app.put("/vehicles/:id", (req, res) => {
-    try {
-        console.log(req.body);
-        const id = req.params.id;
-        const modelName = req.body.modelName;
-        const registrationNumber = req.body.registrationNumber;
-        const category = req.body.category;
+app.get("/vehicles", listVehiclesHandler)
+
+app.put("/vehicles/:id", updateVehiclesHandler)
 
-        connection.query('UPDATE vehicles SET modelName=?, registrationNumber=?, category=? WHERE id =? AND active = 1', [modelName, registrationNumber, category, id], (err, result) => {
-            if (err) throw err;
-            res.status(200).json(result);
-        });
-    }
-    catch (e) {
-        console.log(e.message)
-        res.status(500).json({ errorMessage: 'An unexpected error occured. Check server logs' });
-    }
-})
-//create
-app.post("/vehicles", (req, res) => {
-    try {
-        const modelName = req.body.modelName;
-        const registrationNumber = req.body.registrationNumber;
-        const category = req.body.category;
+app.post("/vehicles", insertVehiclesHandler)
 
-        connection.query('INSERT INTO vehicles(modelName,registrationNumber,category) values(?,?,?)',[modelName, registrationNumber, category], (err, result) => {
-            if (err) throw err;
-            res.status(200).json(result);
-        });
-    }
-    catch (e) {
-        console.log(e.message)
-        res.status(500).json({ errorMessage: 'An unexpected error occured. Check server logs' });
-    }
-})
-//delete
-app.delete('/vehicles/:id', (req, res) =>{
-    try {
-        const id=req.params.id;
-        console.log(req.query.id);
-        connection.query('UPDATE vehicles SET active=0 WHERE id=?',[id], (err, result) =>{
-            if (err) throw err;
-            res.status(200).json(result);
+app.delete('/vehicles/:id', deleteVehiclesHandler)
 
-        });
-    }
-    catch (e) {
-        console.log(e.message)
-        res.status(500).json({ errorMessage: 'An unexpected error occured. Check server logs' });
-    }
-})
\ No newline at end of file
+
+app.listen(port,() => {
+    console.log("port no:5000");
+    })
```

Comment:
Format this file if not already formatted. Also, A better log would be "Listening on port 5000").